examples/LaserWakefield/include/simulation_defines/param/species.param

```diff
@@ -40,6 +40,13 @@
 
 #define ENABLE_IONS PARAM_IONS
 
+/*enable (1) or disable (0) ionization*/
+#ifndef PARAM_IONIZATION
+#define PARAM_IONIZATION 0
+#endif
+
+#define ENABLE_IONIZATION PARAM_IONIZATION
```

Comment:
As already explained, enable_ macros are default controlers and always required. Param_ macros are example-specific.